### PR TITLE
kubectl-gadget/version: Use appropiate error message

### PR DESCRIPTION
**Pull request by burak-ok**
Fixes the usage of the wrong error message function in https://github.com/inspektor-gadget/inspektor-gadget/pull/977#discussion_r988874555

### CODE REVIEW

File: cmd/kubectl-gadget/version.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ var versionCmd = &cobra.Command{
 		opts := metav1.ListOptions{LabelSelector: "k8s-app=gadget"}
 		pods, err := client.CoreV1().Pods("gadget").List(context.TODO(), opts)
 		if err != nil {
-			return commonutils.WrapInErrListNodes(err)
+			return commonutils.WrapInErrListPods(err)
```
Comment (blanquicet) [inline cmd/kubectl-gadget/version.go:54]: That's not the only place where it is wrongly used:

```bash
$ grep "CoreV1().Pods" -C 2 cmd/ -rn
cmd/kubectl-gadget/utils/trace.go-230-  opts := metav1.ListOptions{LabelSelector: "k8s-app=gadget"}
cmd/kubectl-gadget/utils/trace.go:231:  pods, err := client.CoreV1().Pods("gadget").List(context.TODO(), opts)
cmd/kubectl-gadget/utils/trace.go-232-  if err != nil {
cmd/kubectl-gadget/utils/trace.go-233-          return commonutils.WrapInErrListNodes(err)
[...]
cmd/kubectl-gadget/utils/exec.go:53:    pods, err := client.CoreV1().Pods("gadget").List(context.TODO(), listOptions)
cmd/kubectl-gadget/utils/exec.go-54-    if err != nil {
cmd/kubectl-gadget/utils/exec.go-55-            return err
[...]
cmd/kubectl-gadget/utils/flags.go:121:                  pods, err := client.CoreV1().Pods(GadgetNamespace).List(context.TODO(), opts)
cmd/kubectl-gadget/utils/flags.go-122-                  if err != nil {
cmd/kubectl-gadget/utils/flags.go-123-                          return commonutils.WrapInErrListNodes(err)
[...]
```

Comment (burak-ok) [reply cmd/kubectl-gadget/version.go:54]: Tried to convert all appropiate places. There are some in `pkg/container-collection` but there i can not import `commonutils`

Comment (blanquicet) [reply cmd/kubectl-gadget/version.go:54]: No, that error should be used only in the CLI.

**Review by blanquicet — APPROVED** [review]
LGTM! Please squash the commits.

**Pull request merged**